.github/workflows/rucio-release-images.yml

```diff
@@ -0,0 +1,41 @@
+name: Rucio Release Image CI
+
+on:
+  push:
+    tags:
+      - "release-*.cms*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [rucio-probes, rucio-daemons, rucio-server]
+      fail-fast: True
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get git tag name
+        id: gittag
+        run: echo "tag=${GITHUB_REF#refs/tags/}" >> "$GITHUB_ENV"
+
+      - name: Get rucio version
+        id: rucioversion
+        run: echo "rucio_version=$( echo ${{ env.tag }} |  grep -Eo '[0-9]+\.[0-9]+\.[0-9]')" >> "$GITHUB_ENV"
```

Comment:
This is going to need to change (I think). Future rucio versions are going to be 32.x rather than 1.32.x. What I don't know is if it's going to be 32.x.y or just 32.x

I think it's fine as is, but we should remember this and adapt when it happens. :-)

Reply:
I will change the regex to include all these cases.